docs/references/sdk/frontend-only.mdx

```diff
@@ -1,7 +1,152 @@
 ---
-description: TODO
+description: A reference for implementing a frontend-only Clerk SDK
 ---
 
 # Frontend-only SDK
 
-TODO
+While [ClerkJS](/docs/references/javascript/overview) can be used in any browser context and framework, realistically users expect to consume its features through the conventions and syntax of their framework of choice. `@clerk/clerk-react` turns ClerkJS into React components, `@clerk/astro` into Astro components, and so on.
```

Comment:
Since we used _ClerkJS_ on that linked page, I used this formatting throughout the page